### ISSUE TITLE
details of training deeplabv3+ on cocovoc and cityscapes with coarse

### ISSUE THREAD

**Issue by tjulyz** [tensorflow/models#6134]
### System information
- **What is the top-level directory of the model you are using**: deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:binary
- **TensorFlow version (use command below)**: tf1.6
- **Bazel version (if compiling from source)**:no
- **CUDA/cuDNN version**:cuda9.0 cudnn7
- **GPU model and memory**: xp(12g)
- **Exact command to reproduce**: no

### Describe the problem
Thanks for the codes!
I have some questions about the detail of the deeplabv3+
(1) can you share the results of deeplabv3+ on voc2012 testset without coco-pretraining?
(2) can you share the results of deeplabv3+ on cityscape testset without pretraining on coarse set? I have not found these results in your paper.
(3) I want to reproduce the coco-pretraining, so could you please share the details such as training iterations, batchsize for each gpu, learning rate for coco-pretraining and voc finetune, and whether have you used other bn methods?
(4) configurations about training on CItyscapes coarse set and finetune on fine set.

Thank you very very much!

### Source code / logs
no

**Comment by Damanpreet** [comment]
hi @tjulyz were you able to replicate the results on COCO dataset? What training configurations did you use?
